string topic_name
string udp_address
int32 udp_port
int32 width
int32 height
int32 fps
string encoding
